GSA/digitalgov-pra
# POAM: April 2024

This PR handles security vulnerabilities for April 2024

- Closes out dependabot pr -> https://github.com/GSA/digitalgov-pra/pull/370
- [Change log](https://github.com/uswds/uswds/releases) for USWDS update.
- [Change log](https://github.com/sass/dart-sass/blob/main/CHANGELOG.md) for SASS update

There are multiple files that needed to be rebuilt based on the new USWDS version and an update to the Pa11y testing script to excluded any pdf urls 

Testing:
- Pull down branch and run `npm install` and ensure no build errors 
- Start local with `npm run start` and visit local and visually inspect site to ensure no regressions
- Local run `npm run test:pa11y` to ensure no accessibility errors

DependenciesDependency | Old | New
-- | -- | --
 USWDS | 3.71 | 3.8.0 
Sass | 1.69.7 | 1.74.1
